packages/utilities/adaptive-ui/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@microsoft/adaptive-ui",
+  "version": "0.1.0",
+  "description": "A collection of design utilities supporting basic styling and Adaptive UI",
+  "type": "module",
+  "main": "./dist/esm/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/fast.git"
+  },
+  "author": {
+    "name": "Microsoft",
+    "url": "https://discord.gg/FcSNfg4"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/microsoft/fast/issues/new/choose"
+  },
+  "homepage": "https://fast.design",
+  "scripts": {
+    "build": "tsc -p ./tsconfig.build.json && yarn doc",
+    "build:tests": "tsc -p ./tsconfig.test.json",
+    "clean:dist": "node ../../../build/clean.js dist",
+    "coverage": "jest --coverage",
+    "doc": "api-extractor run --local",
+    "doc:ci": "api-extractor run",
+    "prepare": "yarn clean:dist && yarn build",
+    "prettier": "prettier --config ../../../.prettierrc --write \"**/*.ts\"",
+    "prettier:diff": "prettier --config ../../../.prettierrc \"**/*.ts\" --list-different",
+    "test": "yarn build:tests && yarn eslint && yarn unit-tests && yarn doc",
+    "eslint": "eslint . --ext .ts",
+    "eslint:fix": "eslint . --ext .ts --fix",
+    "unit-tests": "mocha",
+    "unit-tests:watch": "mocha --watch"
+  },
+  "jest": {
+    "collectCoverage": true,
+    "coverageReporters": [
+      "json",
+      "text",
+      [
+        "lcov",
+        {
+          "projectRoot": "../../../"
+        }
+      ]
+    ],
+    "coverageThreshold": {
+      "global": {
+        "statements": 72,
+        "branches": 57,
+        "functions": 67,
+        "lines": 72
+      }
+    },
+    "rootDir": "./",
+    "transform": {
+      ".(ts|tsx)": "ts-jest"
```

Comment:
I don't think we need tsx denoted here or ts-jest anymore do we (we aren't using any react here)? I saw the note on Jest breaking, I wonder if we can eject from using that if we don't need to here for tests. Jest doesn't play super nicely and I believe we discussed standardizing on Playwright (@janechu, @nicholasrice, @EisenbergEffect - keep me honest please :) ).

Reply:
Yeah I think this whole `jest` block can be removed. I believe all of these tests were converted to mocha/chai.

Reply:
Is the coverage check different than the tests? @awentzel wanted to make sure we were pulling coverage information for this package. I don't have context here, so I'll do whatever anyone tells me to. :)

Reply:
We can collect coverage but we should pull any jest and Ts-jest dependencies

Reply:
> We can collect coverage but we should pull any jest and Ts-jest dependencies

I'm not sure what that means technically. I see `collectCoverage` under the `jest` section and the `coverage` script calls `jest`. I don't see how to collect coverage without jest dependencies.

I'm just going to pull it all for now to remove the blocker, and it sounds like maybe we have some more cleanup to do across packages that someone who understands what we need can address.